packages/webdriverio/src/scripts/resq.js

```diff
@@ -0,0 +1,55 @@
+export const waitToLoadReact = function waitToLoadReact () {
+    window.resq.waitToLoadReact()
+}
+
+export const react$ = function react$ (selector, props, state) {
+    let elems = window.resq.resq$$(selector)
+
+    /**
+     * see https://github.com/baruchvlz/resq/issues/19
+     */
+    if (typeof elems === 'string') {
+        throw new Error(elems)
```

Comment:
I'll update this in the next patch along side the update to the filtering methods so you don't have to deal with it here.